config/np_network_internode/README

```diff
@@ -0,0 +1,5 @@
+Run dp_network_internode individually using the following command:
+cd ../../; ./run.sh -t "np_network_internode" -r <run-tag> -o <output-dir>
```

Comment:
Lets call this dp_netperf_internode to keep it consistent with respect to naming the data plane and control plane workloads.

Reply:
Done.